code/game/turfs/space/space.dm

```diff
@@ -40,40 +40,27 @@
 		return
 	var/turf/below = GetBelow(src)
 
-
-
 	if(istype(below, /turf/space))
 		return
 	var/area/A = below.loc
 
 	if(!below.density && (A.area_flags & AREA_FLAG_EXTERNAL))
 		return
 
-	//We found a solid thing below us
-	first_solid_z_below = z-1
```

Comment:
this functionality was added for meteors, it was always a hack, but zmimic needs to replace it

Reply:
what have you done to replace this?

Reply:
> what have you done to replace this?

No. I don't replaced this.

Reply:
Meteors use this var to determine what Zlevel to hit. The thing you've deleted here breaks it. Please look into how it works and fix